python/afdko/autohint.py

```diff
@@ -448,12 +448,18 @@
 from afdko.beztools import *
 import traceback
 import shutil
+import warnings
 
 try:
     from psautohint import AUTOHINTEXE
 except ImportError:
     AUTOHINTEXE = "autohintexe"
 
+warnings.warn(
+	"autohint has been deprecated and will be removed from AFDKO soon. "
+	"Please update your code.",
```

Comment:
What about `Please update your code to use psautohint.`?

Reply:
Good idea, done in https://github.com/adobe-type-tools/afdko/pull/1029/commits/6f57a3b5c43863c2c748f34362d5fbe6344e04bd. Also fixed that duplicate import that LGTM/Semmle picked up.